### PR TITLE
Quote, "WYCI" - Ate without table moodlet.

### PR DESCRIPTION
**Pull request by nik707**
## About The Pull Request

Adds a moodlet for eating without a table.
Because Bhijn said WYCI.

## Why It's Good For The Game

i don't know lmao he said wyci and that's basically like saying no balls

## Changelog
:cl:
add: Rimworld
/:cl:

### CODE REVIEW

**Comment by Keekenox** [comment]
I love it.

**Comment by shellspeed1** [comment]
Ate without a table. Time to go crazy and kill the captain

**Comment by nik707** [comment]
rimworld moment

**Review by deathride58 — APPROVED** [review]

**Comment by keronshb** [comment]
This seems like a really weird nerf for no reason.  What about stuff like junk food?

**Comment by bunny232** [comment]
Should only apply if you have the snob quirk.

**Comment by Linzolle** [comment]
rimworld moment

File: code/modules/food_and_drinks/food.dm [hunk 1]
```diff
@@ -49,4 +49,9 @@
 					H.adjust_disgust(25 + 30 * fraction)
 			if((foodtype & BREAKFAST) && world.time - SSticker.round_start_time < STOP_SERVING_BREAKFAST)
 				SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "breakfast", /datum/mood_event/breakfast)
+			if(locate(/obj/structure/table) in range(1, get_turf(H)))
+				to_chat(H,"<span class='notice'>I love tables!</span>")
```
Comment (YakumoChen) [inline code/modules/food_and_drinks/food.dm:53]: I don't think having another to_chat every time you eat near a table is entirely needed.

**Comment by Iamgoofball** [comment]
download

**Comment by Iamgoofball** [comment]
Why?

**Pull request closed without merging**